components/local-app/pkg/auth/auth.go

```diff
@@ -164,8 +163,9 @@ func Login(ctx context.Context, opts LoginOpts) (token string, err error) {
 	}
 
 	defer func() {
-		if closeErr := rl.Close(); closeErr != nil {
-			logrus.WithField("port", port).WithError(closeErr).Debug("Failed to close listener")
+		closeErr := rl.Close()
+		if closeErr != nil {
+			slog.Debug("Failed to close listener", "port", port, "err", closeErr)
```

Comment:
This will change how logging looks like for the local companion, where we don't have a custom slog formatter. Most likely acceptable tho